fix/epsilon/ell1gen.fix.d

```diff
@@ -253,10 +256,27 @@ void Compile(Input input, bool info_, bool verbose, bool write)
         S.Init(input);
         S.Get(Tok);
         $(V1);
+        if (!ignoreContentBeforeEof)
+            recognizeEof();
         $
     }
 }
 
+private void recognizeEof() 
+{
+    while (Tok != endTok) 
+    {
+        if (Tok == sepTok) 
+            continue;
+        else {
```

Comment:
Use Allman style!

BTW: avoid the strange loop:

````
while (Tok == sepTok)
    S.Get(Tok);
if (Tok != endTok)
{
    ++ErrorCounter;
    error!"syntax error, unexpected content before end of file\n%s"(S.Pos);
}
````

Reply:
That's from my Java background... I will do.